### PR TITLE
Add individual plot to profile page.

### PR DESCRIPTION
**Pull request by eschlange**
* Replaced previous profile page overview with garden.
* Added flower images to relate to incentives.
* Added javascript to dynamically update flower positions.
* Updated controllers to include complete incentives.
* Updated existing controller specs and added new ones.
* Updated routes for new plot position update.

[#92071678] [#91981324] [#91981292]

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -14,4 +14,5 @@
 //= require jquery_ujs
 //= require turbolinks
 //= require jquery.toaster
+//= require jquery-ui.min
```
Comment (michaelwehrley) [inline app/assets/javascripts/application.js]: do you need to include `jquery-ui.min` this in both the app and the engine?

File: app/assets/javascripts/plot_positions.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  this.patch_to_server = function(plot_position_id, x_coordinate_percentage, y_coordinate_percentage) {
+    $.ajax({
+      type: "PATCH",
+      url: "/plot_positions/" + plot_position_id,
+      data: {
+        "x": x_coordinate_percentage,
+        "y": y_coordinate_percentage
+      },
+      success: function (data, status) {
+        if ('success' === status) {
```
Comment (michaelwehrley) [inline app/assets/javascripts/plot_positions.js]: Are we doing something for errors?

File: app/assets/stylesheets/sunnyside.css.scss [hunk 3]
```diff
@@ -13,4 +15,36 @@ a.navbar-brand {
...
+  height: 600px;
+
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: .75em;
+  margin-bottom: .75em;
+
+  box-shadow: 0 0 5px green;
+
+  border-radius: 1em!important;
```
Comment (michaelwehrley) [inline app/assets/stylesheets/sunnyside.css.scss]: I will take your expertise on the need for this `!important` :smile:

File: app/views/social_networking/profile_pages/show.html.erb [hunk 4]
```diff
@@ -31,23 +31,32 @@
...
-    </div>
 
-    <div class="profiles-last-online profile-box-border">
-      <h3>Last seen: {{ profile.profile.latestAction | timeFromNow }}</h3>
+      <div class="profiles-last-online profile-box-border">
+        <h3>Last seen: {{ profile.profile.latestAction | timeFromNow }}</h3>
+      </div>
+
+      <div id="garden-individual" class="garden profile-page-garden">
+        <% @profile
```
Comment (michaelwehrley) [inline app/views/social_networking/profile_pages/show.html.erb]: Not sure it really makes big deal here, but could/should these chained methods be in the controller?

**Comment by michaelwehrley** [comment]
Great work!  While I have some questions, nothing that should keep you from moving forward. :smile:

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (eschlange) [inline app/assets/javascripts/application.js]: Mike, good question... I probably only need it in the engine JS, but I'll test it out.

File: app/assets/javascripts/plot_positions.js (hunk 2 above)
Comment (eschlange) [inline app/assets/javascripts/plot_positions.js]: What sort of errors?

File: app/assets/stylesheets/sunnyside.css.scss (hunk 3 above)
Comment (eschlange) [inline app/assets/stylesheets/sunnyside.css.scss]: Copied from Mark's CSS. He is a CSS master~

File: app/views/social_networking/profile_pages/show.html.erb (hunk 4 above)
Comment (eschlange) [inline app/views/social_networking/profile_pages/show.html.erb]: I should probably refactor the scope into the model and the html into its own partial. I'll begin work on this now.

File: app/assets/javascripts/plot_positions.js (hunk 2 above)
Comment (eschlange) [inline app/assets/javascripts/plot_positions.js]: I see what you are saying, I'm going to vote for silent failures to update the garden image positions. This should only happen when the server is down, and if the server is down, then a page refresh would result in error as well. Would you have any recommendations on a better user experience?

**Pull request merged**